src/tsconfig.json

```diff
@@ -3,9 +3,9 @@
 	"compilerOptions": {
 		"rootDir": "./",
 		"outDir": "../dist",
-		"composite": true
+		"composite": true,
+		"preserveConstEnums": true,
```

Comment:
```suggestion
```

That's a default option

https://github.com/sapphiredev/utilities/blob/b2b662e1a64720ccc8a2e11709fdd0bbeedd95e5/packages/ts-config/src/tsconfig.json#L24